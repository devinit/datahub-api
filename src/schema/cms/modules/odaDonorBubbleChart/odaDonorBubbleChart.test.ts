import {getOdaDonorBubbleChartPageData} from '.';
import * as prettyFormat from 'pretty-format';

describe('ODA donor bubble chart module tests', () => {
    it('should return page data for the ODA donor bubble chart', async () => {
        const pageData: DH.IPage[] = await getOdaDonorBubbleChartPageData();
      //  console.log(pageData[0]);
        expect(pageData.length).toBeGreaterThan(0);
        expect(prettyFormat(pageData)).toMatchSnapshot();
    }, );
});
