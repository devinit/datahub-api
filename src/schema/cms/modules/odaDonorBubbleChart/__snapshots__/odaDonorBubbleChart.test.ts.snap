// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ODA donor bubble chart module tests should return page data for the ODA donor bubble chart 1`] = `
"Array [
  Object {
    \\"id\\": \\"diff-providers-priorities\\",
    \\"meta\\": \\"\\",
    \\"narrative\\": \\"How effectively are donors targeting the countries with the highest numbers of people in poverty and the least domestic public resources to address it? This chart allows you to select a provider of aid to see how it allocates aid across countries\\",
    \\"title\\": \\"Different providers, different priorities\\",
  },
]"
`;
